helm/values.yaml

```diff
@@ -6,6 +6,7 @@ superblocks: {}
   # agentKey: ""
   # agentHostUrl: "http[s]://<agent-host[:port]>/agent"
   # agentEnvironment: <"*"|"staging"|"production">
+  # agentDataDomain: <"app.superblocks.com"|"eu.superblocks.com">
```

Comment:
Deferrable - let’s rely on this for __SUPERBLOCKS_AGENT_SERVER_URL as well

Reply:
__superblocks_Agent_server_url will be fixed to `api.superblocks.com` as that is the global accelerator backing the edge caches